src/StackExchange.Redis/RedisResult.cs

```diff
@@ -401,7 +401,7 @@ internal override bool AsBoolean()
                     : Array.ConvertAll(_value, x => x.AsByteArray()!);
 
             private bool IsSingleton => _value?.Length == 1;
-            private bool IsEmpty => _value?.Length == 0;
```

Comment:
this changes the value of `IsEmpty` for `null` values of `_value`; currently a `null` will return `false` for `IsEmpty`; with this change it will return `true`. The question is what *should* `IsEmpty` return for `null` values? IMO we should retain the current behaviour, where `null` reports `false` here (note the separate `IsNull` test). Changing the result of this property is a significant proposition that would require justification.